index.html

```diff
@@ -224,5 +167,40 @@ <h2 class="meetup-date">24 nov</h2>
     </div>
   </footer>
 </body>
+<script>
+fetch('https://api.github.com/repos/devfsa/meetups/issues/3/comments')
+  .then((response)=> {
+      response.json().then((data)=> {
+        data = data.slice(0,4)
+        let html = '';
+        let keynotes = document.getElementById("keynotes");
+        data.forEach(element => {
+          let { user, html_url, body } = element
+          let title = body.split('\r\n')[0].replace('**', '').replace('**', '')
+          let description = body.split('\r\n')[1]
+          html += '<div class="card keynote-card">'
+          html += '<div class="keynote-image"><img src="' + user.avatar_url + '" class="keynote-photo"></div>'
+          html += '<div class="keynote-desc">'
+          html += '<a href="' + html_url + '" class="keynote-title" target="_blank">' + title + '</a>'
+          html += '<div class="keynote-info">'
+          html += '<p class="keynote-author">@' + user.login + '</p>'
+          html += '<a href="' + html_url + '" class="keynote-vote voted" target="_blank">'
+          html += '<i class="far fa-arrow-alt-circle-up"></i>'
+          html += '<span class="keynote-vote-number"></span>'
+          html += '</a>'
+          html += '</div>'
+          html += '</div>'
+          html += '</div>'
+        });
+        console.log(html)
+        keynotes.innerHTML = html
```

Comment:
A suggestion based on [this stackoverflow question](https://stackoverflow.com/questions/2305654/innerhtml-vs-appendchildtxtnode) will improve performance:
```suggestion
        keynotes.insertAdjacentHTML('beforeend', html);
```